### PR TITLE
Ability to undo automatic changes to current path.

### PR DESCRIPTION
**Pull request by Osmose**
Based on feedback from @bwinton in #3.

### CODE REVIEW

**Comment by Osmose** [comment]
@bwinton r?

File: lib/advanced-open-file-view.coffee [hunk 1]
```diff
@@ -270,29 +272,38 @@ class AdvancedFileView extends View
 
     # Handle keyboard movement
     @miniEditor.on 'keydown', (ev) =>
-      if ev.keyCode is 9
+      if ev.keyCode is 9 # Tab
         blockEvent ev
         return
 
+      if ev.keyCode is 90 and (ev.ctrlKey or ev.metaKey) # Ctrl/Cmd + Z
```
Comment (bwinton) [inline lib/advanced-open-file-view.coffee]: So, this makes ctrl-z work on Mac, which is a little weird, but probably hard to fix…

**Comment by bwinton** [comment]
Yeah, other than that, it seems pretty good to me.  :shipit:

File: lib/advanced-open-file-view.coffee (hunk 1 above)
Comment (Osmose) [inline lib/advanced-open-file-view.coffee]: Yeah, I figured the oddity wasn't worth fixing. In reality I'd like this to either pull from whatever atom has for an undo shortcut, or be configurable, but neither of those seemed easy to do, so I figured I'd go with the stupid solution and wait for someone to complain. :D

**Comment by Osmose** [comment]
Thanks for the review!

**Pull request merged**